src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_ArrayCreationAndInitializer.vb

```diff
@@ -1206,7 +1206,7 @@ IVariableDeclarationStatement (1 declarations) (OperationKind.VariableDeclaratio
     Variables: Local_1: x As System.Int32()
     Initializer: 
       IVariableInitializer (OperationKind.VariableInitializer, IsImplicit) (Syntax: 'x(2)')
-        IArrayCreationExpression (OperationKind.ArrayCreationExpression, Type: System.Int32()) (Syntax: 'x(2)')
+        IArrayCreationExpression (OperationKind.ArrayCreationExpression, Type: System.Int32(), IsImplicit) (Syntax: 'x(2)')
```

Comment:
It feels a little weird to me that the ArrayCreation is the implicit node. I'd expect the VariableInitializer to be the implicit one.

Reply:
> It feels a little weird to me that the ArrayCreation is the implicit node. I'd expect the VariableInitializer to be the implicit one.

I believe all nodes are supposed to be implicit here except the size of the array. There is no code that explicitly creates an array.